conf.d/health.d/ipfs.conf

```diff
@@ -0,0 +1,11 @@
+
+   alarm: ipfs_datastore_in_use
+      on: ipfs_local.repo_size
```

Comment:
I suggest to change this to `template` in `ipfs.repo_size`, so that it will be applied to all ipfs instances.